packages/ui5-middleware-livereload/lib/livereload.js

```diff
@@ -67,13 +67,31 @@ module.exports = async ({ resources, options }) => {
     if (options.configuration && options.configuration.debug) {
         debug = options.configuration.debug;
     }
+        
+    let serverOptions = {
+        debug: debug,
+        extraExts: extraExts ? extraExts.split(",") : undefined,
+        port: port,
+        exclusions: exclusions
+    };
+    
+    const cli = require("yargs");
+    if (cli.argv.h2) {
+        const path = require("path");
+        const os = require("os");
+        const fs = require("fs");
+        
+        sslKeyPath = cli.argv.key ? cli.argv.key : path.join(os.homedir(), ".ui5", "server", "server.key");
+        sslCertPath = cli.argv.cert ? cli.argv.cert : path.join(os.homedir(), ".ui5", "server", "server.crt");
+        
```

Comment:
please provide a `log.debug(...)` on where the runtime retrieves the certs from - convenient for the user when things break

Reply:
Debug info added.